cms/templates/index.html

```diff
@@ -467,6 +472,44 @@ <h4 class="title">${_('Your Course Creator Request Status:')}</h4>
       </div>
       % endif
 
+      %if archived_courses:
+      <div class="archived-courses archived-courses-tab">
```

Comment:
Oh great, a third copy of this HTML. :(

I'm trying to remove this duplication in #15390. It shouldn't be a big deal to merge with these changes though.